lighthouse-core/audits/content-width.js

```diff
@@ -6,6 +6,18 @@
 'use strict';
 
 const Audit = require('./audit.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  title: 'Content is sized correctly for the viewport',
+  failureTitle: 'Content is not sized correctly for the viewport',
+  description: 'If the width of your app\'s content doesn\'t match the width ' +
+      'of the viewport, your app might not be optimized for mobile screens. ' +
+      '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/content-sized-correctly-for-viewport).',
+  explanation: 'The viewport size is {innerWidth}px, whereas the window size is {outerWidth}px.',
```

Comment:
can we deflower the language here - `whereas` -> `and`.

Reply:
ya, I admit I didn't really parse the sentence.

Reply:
whereas was written by a british person (#493), seems fine to keep :)

Reply:
Reworded to remove all confusion.  @brendankenny that seems like en-GB and we write these in en-US 🇬🇧

Reply:
well maybe if you all picked up a book once in a while. At least get your 🇺🇸 commas right, then

Reply:
,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,whereas,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,how,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,about,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,no,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,

Reply:
ha, no, I mean fix the comma in your new string :P